tests/codex/testerasure.nim

```diff
@@ -22,25 +22,13 @@ asyncchecksuite "Erasure encode/decode":
   var manifest: Manifest
   var store: BlockStore
   var erasure: Erasure
-  var repoDs: Datastore
-  var metaDs: SQLiteDatastore
 
   setup:
     rng = Rng.instance()
     chunker = RandomChunker.new(rng, size = dataSetSize, chunkSize = BlockSize)
-    manifest = !Manifest.new(blockSize = BlockSize)
-    repoDs = SQLiteDatastore.new(Memory).tryGet()
-    metaDs = SQLiteDatastore.new(Memory).tryGet()
-    store = RepoStore.new(repoDs, metaDs)
+    store = CacheStore.new(cacheSize = (dataSetSize * 8), chunkSize = BlockSize)
```

Comment:
I would not use the a different store from the repostore here (or elsewhere). I understand the desire to isolate the tests as much as possible, but the more we rely on concrete components instead of mocks, the more robust our tests are going to be. In this case, the repostore is perfectly usable with an in-memory sql and doesn't require any mocking.

Reply:
Ok, I agree. I'm going to use the same type of stores that were used before.